lib/web_ui/test/text/measurement_test.dart

```diff
@@ -581,17 +768,25 @@ void main() async {
         // This can only be done correctly in the canvas-based implementation.
         if (instance is CanvasTextMeasurementService) {
           expect(result.height, 20);
+
+          expect(result.lines, <EngineLineMetrics>[
+            line('abcdef ', hardBreak: false, width: 60.0, lineNumber: 0),
```

Comment:
Do we have a mode where we measure with whitespace? I haven't seen any tests that test for that. I imagine we'd need that for text fields?

Reply:
`LineMetrics` doesn't include measurement of trailing whitespace. The only thing that measures trailing whitespace is `maxIntrinsicWidth` and there are tests for that.

Reply:
👍